apollo-router/src/plugins/telemetry/apollo.rs

```diff
@@ -35,59 +34,72 @@ pub(crate) const ENDPOINT_DEFAULT: &str =
 
 pub(crate) const OTLP_ENDPOINT_DEFAULT: &str = "https://usage-reporting.api.apollographql.com";
 
-#[derive(Derivative)]
-#[derivative(Debug)]
-#[derive(Clone, Deserialize, JsonSchema)]
-#[serde(deny_unknown_fields, default)]
+#[derive(Clone, Deserialize, JsonSchema, Debug)]
+#[serde(deny_unknown_fields)]
 pub(crate) struct Config {
     /// The Apollo Studio endpoint for exporting traces and metrics.
     #[schemars(with = "String", default = "endpoint_default")]
+    #[serde(default = "endpoint_default")]
     pub(crate) endpoint: Url,
 
     /// The Apollo Studio endpoint for exporting traces and metrics.
     #[schemars(with = "String", default = "otlp_endpoint_default")]
+    #[serde(default = "otlp_endpoint_default")]
     pub(crate) experimental_otlp_endpoint: Url,
 
     /// The Apollo Studio API key.
     #[schemars(skip)]
-    #[serde(skip)]
+    #[serde(default = "apollo_key")]
     pub(crate) apollo_key: Option<String>,
 
     /// The Apollo Studio graph reference.
     #[schemars(skip)]
-    #[serde(skip)]
+    #[serde(default = "apollo_graph_reference")]
     pub(crate) apollo_graph_ref: Option<String>,
 
     /// The name of the header to extract from requests when populating 'client nane' for traces and metrics in Apollo Studio.
     #[schemars(with = "Option<String>", default = "client_name_header_default_str")]
```

Comment:
Is it still an Option in schemars? I find that odd anyway with a default, but that was already there I guess...

Reply:
Please raise that issue.

Reply:
https://github.com/apollographql/router/issues/3982